packages/react-mutation-mapper/src/util/LollipopPlotUtils.spec.ts

```diff
@@ -2,12 +2,23 @@ import { assert } from 'chai';
 
 import { Mutation } from 'cbioportal-utils';
 
-import { lollipopLabelText } from './LollipopPlotUtils';
+import {
+    byMiddlemostPosition,
+    byMutationCount,
+    byWhetherExistsInADomain,
+    calcCountRange,
+    calcYMaxInput,
+    getCommonYAxisMaxSliderValue,
+    getYAxisMaxInputValue,
+    getYAxisMaxSliderValue,
+    lollipopLabelText,
+    lollipopLabelTextAnchor,
+} from './LollipopPlotUtils';
 
 describe('LollipopPlotUtils', () => {
     let mutationsAtPosition: Mutation[];
 
-    beforeAll(() => {
+    before(() => {
```

Comment:
IDEs sometimes get confused about which test methods belongs to which test suite. I think this is because we use jest for packages, but mocha for the main project. I would leave this as is.